### PR TITLE
update start_bot.py and add a script to generate new tokens

### PR DESCRIPTION
**Pull request by sebag90**
update the `start_bot.py`:
- allow custom naming of the concierge bot if the concierge bot is being started alone
- add the waiting room id to the environment variables of the concierge bot for easier identification through the `docker inspect` command

add the `generate_tokens.py` script to generate new tokens for a bot that is already running

### CODE REVIEW

File: start_bot.py [hunk 1]
```diff
@@ -446,13 +452,13 @@ def main(args):
         config.read(Path(args.credentials_from_file))
         config.sections()
 
-        if any(config["SLURK CREDENTIALS"].get(i) is None for i in ["host", "token"]):
+        if any(config["SLURK"].get(i) is None for i in ["host", "token"]):
             raise ValueError("Invalid formatting for credentials file")
 
-        sulrk_address = config.get("SLURK CREDENTIALS", "host")
+        sulrk_address = config.get("SLURK", "host")
```
Comment (janagoetze) [inline start_bot.py:458]: let's fix the spelling

File: generate_tokens.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        config.sections()
+
+        if any(config["SLURK"].get(i) is None for i in ["host", "token"]):
+            raise ValueError("Config file is missing slurk entries")
+
+        if any(config["BOT"].get(i) is None for i in ["task_id", "waiting_room_id"]):
+            raise ValueError("Config file is missing slurk entries")
+
+        sulrk_address = config.get("SLURK", "host")
+        SLURK_HOST = sulrk_address
```
Comment (janagoetze) [inline generate_tokens.py:138]: slurk_address

File: start_bot.py (hunk 1 above)
Comment (sebag90) [reply start_bot.py]: :+1:

File: generate_tokens.py (hunk 2 above)
Comment (sebag90) [reply generate_tokens.py]: :+1:

**Pull request merged**